pkg/provider/fake/fake.go

```diff
@@ -266,7 +249,7 @@ func mapKey(key, version string) string {
 }
 
 func init() {
-	esv1beta1.Register(&Provider{}, &esv1beta1.SecretStoreProvider{
-		Fake: &esv1beta1.FakeProvider{},
-	})
+	esv1.Register(&Provider{}, &esv1.SecretStoreProvider{
+		Fake: &esv1.FakeProvider{},
+	}, esv1.MaintenanceStatusNotMaintained) // TODO - update this post tests
```

Comment:
I actually like having a fake store that is unmaintained. Maybe we should leave it? If not:
```suggestion
	}, esv1.MaintenanceStatusMaintained)
```